moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_path_constraints.cpp

```diff
@@ -112,24 +115,19 @@ class FixStartStatePathConstraints : public planning_request_adapter::PlanningRe
           // we need to append the solution paths.
           res2.trajectory_->append(*res.trajectory_, 0.0);
           res2.trajectory_->swap(*res.trajectory_);
-          return true;
+          return moveit_msgs::MoveItErrorCodes::SUCCESS;
         }
-        else
-          return false;
       }
-      else
+      if (!bool(solved1) || !bool(solved2))
       {
-        ROS_WARN("Unable to plan to path constraints. Running usual motion plan.");
-        bool result = planner(planning_scene, req, res);
-        res.planning_time_ += res2.planning_time_;
-        return result;
+        ROS_WARN("Unable to meet path constraints at the start.");
+        moveit::core::MoveItErrorCode moveit_code(moveit_msgs::MoveItErrorCodes::START_STATE_VIOLATES_PATH_CONSTRAINTS);
+        res.error_code_.val = moveit_msgs::MoveItErrorCodes::START_STATE_VIOLATES_PATH_CONSTRAINTS;
+        return moveit_code;
```

Comment:
Nit: Would that work as well?
```suggestion
        res.error_code_.val = moveit_msgs::MoveItErrorCodes::START_STATE_VIOLATES_PATH_CONSTRAINTS;
        return moveit::core::MoveItErrorCode(res.error_code_.val);
```

Reply:
Yep, nice simplification